lib/SILOptimizer/Mandatory/MandatoryInlining.cpp

```diff
@@ -663,15 +663,6 @@ class MandatoryInlining : public SILModuleTransform {
                                SetFactory, SetFactory.getEmptySet(), CHA);
     }
 
-    // Make sure that we de-serialize all transparent functions,
```

Comment:
I just want to double check. What was the commit that fixed this?

Reply:
It wasn't a "fix" per se, just a removal of an unsound optimization. See f10f8d55dc43ad1addfc5e4854f691e0d0099049.